tests/diagnostics/uninitialized-out-parameters.slang

```diff
@@ -47,5 +47,48 @@ void control_flow_defined(bool b, out float y)
     return;
 }
 
+// Specialized handling for intrinsic asm
+void instrincs_defined(float x, out float exp)
+{
+    __intrinsic_asm "frexp";
+}
+
+// Specialized handling for target switches
+void target_switching_undefined(float x, out float exp)
+{
+    //CHK-DAG: warning 41018: returning without initializing out parameter 'exp'
+    __target_switch {}
```

Comment:
Using `-DAG` looks dangerous, because it may match to a warning from totally different part of this test.
You can include the line number into the pattern.